libraries/botframework-connector/tests/auth/managedIdentityAppCredentials.test.js

```diff
@@ -0,0 +1,99 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+const assert = require('assert');
+const sinon = require('sinon');
+const { JwtTokenProviderFactory } = require('../../lib/auth/jwtTokenProviderFactory');
+const { ManagedIdentityAppCredentials } = require('../../lib/auth/managedIdentityAppCredentials');
+const { AppCredentials } = require('../../lib/auth/appCredentials');
+const { AuthenticationConstants } = require('../../lib/auth/authenticationConstants');
```

Comment:
```suggestion
const { AppCredentials, AuthenticationConstants, JwtTokenProviderFactory, ManagedIdentityAppCredentials } = require('../../lib');
```

Note: this will only work once `ManagedIdentityAppCredentials` are publicly accessible.